main.py

```diff
@@ -261,16 +261,85 @@ def get_last_meeting_date(self):
                 date = previous_date.date() + datetime.timedelta(days=1)
         return date
 
+    def get_meeting_settings(self):
+        """Load Zoom meeting settings"""
+        table_name = "Zoom_Meeting_Settings"
+        meeting_ids = self._get_meeting_ids()
+        total_settings = 0
+        for meeting_id in meeting_ids:
+            page_token = True
+            params = {"id": meeting_id, "page_size": 300, "type": "past"}
+            results = []
+            while page_token:
+                response = self.client.meeting.get(**params).json()
+                if response.get("code") == 429:
+                    logging.info("Rate limit reached; waiting 10 seconds...")
+                    time.sleep(10)
```

Comment:
is this waiting but not trying again? I don't see any retry logic

Reply:
Because the `_get_meeting_ids()` method does a diff with the possible meetings, any skipped meeting will get picked up on subsequent runs. This was a shortcut to just slow down the requests rather than actually retry the request which would require more complexity or patching the zoomus library to do request retries.